### ISSUE TITLE
Place holder for dead/missing germplasm [REQUEST]: 

### ISSUE THREAD

**Issue by SCYarnes** [PhenoApps/Field-Book#909]
### Description

Breeders of long-lived species will often need to navigate dead/missing germplasm positions when they phenotype. It is important to include placeholders for positions while data collecting without offering the ability to accidentally collect phenotypes for these missing positions.

Below is a field map from a pecan orchard. Within the contiguous set of trees there are lots of positions with missing trees. The breeder would like to see placeholders for these positions within Field Book. The current work around with DeltaBreed is to use a dummy germplasm record, but this allows for the accidental association of phenotypes on these positions. We would like to be able to associate a position with 'NA' germplasm, and disallow data collection on these positions.
![image](https://github.com/PhenoApps/Field-Book/assets/142930813/3540854e-a744-4d0b-935a-975ee6dcd84b)

**Comment by SCYarnes** [comment]
I don't have a good sense of the best way to implement this request, but if a solution is implemented in FB, we will recapitulate the solution within DeltaBreed. I only prefer that the solution not originate from NAs in plot ID.

**Comment by keocorak** [comment]
Breedbase is having a similar convo over here [https://github.com/solgenomics/sgn/issues/4881](url)

**Comment by trife** [comment]
@SCYarnes Would removing the entry from being visible in Collect solve this issue?

**Comment by SCYarnes** [comment]
@trife No, I don't think so. They specifically format their FB files to include "skip" so that they will scroll though the dead/missing positions from the collect screen. They just don't want to be prompted to accidentally enter any observations for those positions.
